plugins/services/src/js/reducers/serviceForm/Containers.js

```diff
@@ -365,8 +370,9 @@ module.exports = {
       switch (type) {
         case ADD_ITEM:
           const name = `container-${newState.length + 1}`;
+          const newItem = Object.assign({}, DEFAULT_POD_CONTAINER, { name });
 
-          newState.push(Object.assign({}, DEFAULT_POD_CONTAINER, { name }));
+          newState.push(value || newItem);
```

Comment:
I'm pretty sure that we never want to add any falsy values but we probably should adjust the line to avoid confusions and clearly state the expected type. Good catch!